server/router/user.js

```diff
@@ -3,5 +3,6 @@ const router = require('express').Router();
 const { logoutController } = require('../controller/routes/user');
 
 router.get('/logout', logoutController);
+router.patch('/users');
```

Comment:
Where does the user ID come from?

Reply:
I take it from token. Should I pass it in params?

Reply:
Yeah that would be more REST-ish. It will work either way, but passing it in the params would be clearer